packages/shared/src/features/annotation/types.ts

```diff
@@ -13,3 +13,92 @@ export type ConfigCategory = z.infer<typeof configCategory>;
 export type CastedConfig = Omit<ScoreConfig, "categories"> & {
   categories: ConfigCategory[] | null;
 };
+
+const ScoreSource = z.enum(["ANNOTATION", "API", "EVAL"]);
+
+const ScoreBase = z.object({
+  id: z.string(),
+  timestamp: z.date(),
+  projectId: z.string(),
+  name: z.string(),
+  source: ScoreSource,
+  authorUserId: z.string().nullish(),
+  comment: z.string().nullish(),
+  traceId: z.string(),
+  observationId: z.string().nullish(),
+  configId: z.string().nullish(),
+  createdAt: z.date(),
+  updatedAt: z.date(),
+});
+
+const NumericData = z.object({
```

Comment:
you could use ScoreBase.extend() here which would reduce the need for merge below and make NumericData usable independently if necessary later

Reply:
I think the current version is more readable, but feel free to disagree :)